packages/wdio-allure-reporter/src/index.js

```diff
@@ -221,8 +223,11 @@ class AllureReporter extends WDIOReporter {
 
         // set hook (test) status
         if (hook.error) {
-            this.onTestFail(hook)
-        } else {
+            if (isMochaAllHooks(hook.title)) {
```

Comment:
is it necessary to start test here? See https://github.com/webdriverio/webdriverio/blob/master/packages/wdio-allure-reporter/src/index.js#L123

Reply:
In my opinion it's more readable. If I remove it then there's onTest Fail or Pass without onTestStart. So we finishing without starting. Thou if you insist I'll remove it, np.

Reply:
ok makes sense